rt106f_smart_lock/board/board_define.h

```diff
@@ -45,8 +46,35 @@
 #define ENABLE_INPUT_DEV_ShellUsb
 #define ENABLE_LPM_DEV_Standby
 #define ENABLE_INPUT_DEV_Lpc845uart
+//#define ENABLE_INPUT_DEV_WiFiAWAM510
```

Comment:
I saw you added so many code related to WIFI module, did you have a test on this function?